huaweicloud/services/acceptance/fgs/data_source_huaweicloud_fgs_quotas_test.go

```diff
@@ -0,0 +1,100 @@
+package fgs
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDataSourceQuotas_basic(t *testing.T) {
+	var (
+		all = "data.huaweicloud_fgs_quotas.test"
+		dc  = acceptance.InitDataSourceCheck(all)
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPreCheckFgsAgency(t)
```

Comment:
It is recommended to add comments to the environment variables used.

Reply:
there is no need to use this env for the function create, in this scenario, the function does not needs the other service's privileges.